docs/integration/kubernetes.md

```diff
@@ -772,6 +772,64 @@ Container and Kubernetes metrics are collected along with labels and tags, which
 |----|-------|--------|-------------------------------------|
 |scheduler.attempts.total_num| gauge |  | number of attempts to schedule pods |
 
+### Kube-controller Metrics
+#### Workqueue Wait
+| Metric Name                              | Type  | Unit    | Description                                          |
+|------------------------------------------|-------|---------|------------------------------------------------------|
+| kubecontroller.workqueue_wait.p50latency | gauge | seconds | p50 latency of waiting time before item is processed |
+| kubecontroller.workqueue_wait.p75latency | gauge | seconds | p75 latency of waiting time before item is processed |
+| kubecontroller.workqueue_wait.p90latency | gauge | seconds | p90 latency of waiting time before item is processed |
+| kubecontroller.workqueue_wait.p95latency | gauge | seconds | p95 latency of waiting time before item is processed |
+| kubecontroller.workqueue_wait.p99latency | gauge | seconds | p99 latency of waiting time before item is processed |
+| kubecontroller.workqueue_wait.duration   | gauge | seconds | duration of waiting time before item is processed    |
+| kubecontroller.workqueue_wait.total_num  | gauge |   | number of items being processed                      |
+
+#### Workqueue Process
+|Metric Name| Type  | Unit    | Description                                            |
+|----|-------|---------|--------------------------------------------------------|
+|kubecontroller.workqueue_process.p50latency| gauge | seconds | p50 latency of item being processed by Kube-controller |
+|kubecontroller.workqueue_process.p75latency| gauge | seconds | p75 latency of item being processed by Kube-controller |
+|kubecontroller.workqueue_process.p90latency| gauge | seconds | p90 latency of item being processed by Kube-controller |
+|kubecontroller.workqueue_process.p95latency| gauge | seconds | p95 latency of item being processed by Kube-controller |
+|kubecontroller.workqueue_process.p99latency| gauge | seconds | p99 latency of item being processed by Kube-controller |
+|kubecontroller.workqueue_process.duration| gauge | seconds | duration of item being processed by Kube-controller    |
+|kubecontroller.workqueue_process.total_num| gauge |   | number items being processed by Kube-controller        |
+
+#### Workqueue Depth
+|Metric Name| Type  | Unit    | Description                                 |
+|----|-------|---------|---------------------------------------------|
+|kubecontroller.workqueue_depth.total_num| gauge |   | number of items pending for being processed |
+
+#### Workqueue Unfinished Work
+|Metric Name| Type  | Unit    | Description                                         |
+|----|-------|---------|-----------------------------------------------------|
+|kubecontroller.workqueue_unfinished_work.total_num| gauge |   | number of items remaining unfinished in a workqueue |
```

Comment:
number of items remaining in the work queue
or simply
number of items in the work queue

Reply:
work queue or workqueue. I assume the former is better.  Just be consistent.